Deliverables/ObligatoriskOppgave3.md

```diff
@@ -0,0 +1,87 @@
+		- Lage tiles
+		- Lage teksturer
+		- Implementere grafikken
+	- **Akseptansekrav**
+		- Om det finnes elementer på brettet så vises disse på det grafiske spillbrettet
+- "Som bruker ønsker jeg å flytte brikker på brettet fordi dette er et interaktivt spill"
+	- **Arbeidsoppgaver**
+		- Implementere logikk for bevegelse
+	- **Akseptansekrav**
+		- Det går an å gi input til spillet slik at spillerbrikken beveger seg på spillbrettet
+- "Som bruker ønsker jeg å kunne besøke flaggene på brettet for å vinne spillet"
+	- **Arbeidsoppgaver**
+		- Implementere visningsgrafikk for flagg
+		- Implementere logikk for besøk av flagg
+		- Implementere logikk for seier når alle flagg er besøkt i riktig rekkefølge
+	- **Akseptansekrav**
+		- Spillerbrikken kan bevege seg til flagg på kartet
+		- Om spillerbrikken besøker alle flaggene i riktig rekkefølge vinner spilleren spillet
+- "Som bruker ønsker jeg å se andre spillere og se dem flytte seg for å kunne spille flerspiller"
+	- **Arbeidsoppgaver**
+		- Implementere logikk for flerspiller
+		- Implementere grafikk for flere spillere
+		- Implementere "turnBase" logikk
+	- **Akseptansekrav**
+		- To brukere kan være aktiv på samme spillbrett og se hverandres brikker i korrekt posisjon
+- "Som bruker ønsker jeg å kunnne bli tildelt kort for å kunne programmere roboten min"
+	- **Arbeidsoppgaver**
+		- Implementere kort
+		- Lage logikk for utdeling av kort
+		- Lage logikk for visning av kort til brukeren
+	- **Akseptansekrav**
+		- Det finnes kort i spillet som kan deles ut til spillerene
+		- Hver spiller kan se kortene de har fått tildelt
+- "Som bruker ønsker jeg å kunnne "spille" et kort for å bevege roboten min"
+	- **Arbeidsoppgaver**
+		- Implementere logikk for valgte kort og bevegelse
+		- Implementere grafikk som viser at robot har blitt flyttet
+	- **Akseptansekrav**
+		- Når en spiller legger et kort, beveger roboten seg med henhold til egenskapene til kortet som ble lagt
+	
+###Deloppgave 3
+- Se README.md om kjøring av spillet og tester
+
+## Retrospective for Oblig 3
+- I denne obligen har vi fokusert på å gjøre ferdig MVP, derav:
+1. Multiplayer, å vise roboter og bevegelser på alle skjermer (ikke bare hos host)
+2. Få til win-condition ved flagg.
+3. Få til loss-condition ved hull.
+- Dette er nå ferdig, og spillet har dermed oppnådd MVP. Dette er vi veldig fornøyde med, siden å få til MVP var hovedmålet vårt for denne obligen.
+- Multiplayer har vært en stor oppgave å få til, så å vise roboter samt få dem til å flytte seg i clients er vi veldg fornøyde med.
+- Vi ser at det har vært en positiv forbedring å delegere oppgaver tidligere i prosessen, slik at vi har bedre og mer effektiv tid til å gjennomføre målet vi har satt oss.
+- Til neste oblig bør vi fokusere mer på jevne commits og jevn arbeidsfordeling. 
+- Nå som vi er ferdig med MVP, kan vi fokusere på mer brukervennlig GUI. Til nå er spillet spilt i terminalen, men vi ønsker å ha en mer interaktiv GUI hvor spiller kan velge kort.  
```

Comment:
Nå som vi er ferdig med MVP, kan vi fokusere på mer brukervennlig GUI. Til nå er spillet spilt i terminalen. Men til neste gang ønsker vi å gjøre det mulig for spilleren å interagere med spillet helt gjennom GUI.  Dette vil være ved å velge kort, se healthtokens og lignende.